src/saltext/vmware/states/esxi.py

```diff
@@ -1680,3 +1680,78 @@ def vsan_config(
         ret["comment"] = "VSAN configuration will change."
 
     return ret
+
+
+def apply_configuration(
+    name, cluster_path, desired_config, show_changes=False, check_compliance=False, profile=None
+):
+    """
+    Apply desired state configuration.
+    """
+
+    # Keep this structure
+    ret = {
+        "name": name,
+        "result": True,
+        "changes": {},
+        "comment": "Configuration applied successfully",
+    }
+
+    config = __opts__
+    esx_config = utils_esxi.create_esx_config(config, profile)
+
+    # CREATE DRAFT
+    draft_create_response = vmware_esxi.draft_create(
```

Comment:
again with draft, a better naming convention should be used here, and what exactly is a draft_create trying to achieve, it is creating something based on imports - hence why is 'create' not good enough given the inputs to the function will specify what is to be generated from the inputs

This applies to all things prepended with 'draft_'